specs/span-batches.md

```diff
@@ -149,11 +150,16 @@ Span-batch rules, in validation order:
   - Rules:
     - `start_epoch_num + sequence_window_size < inclusion_block_number` -> `drop`:
       i.e. the batch must be included timely.
-    - `end_epoch_num < epoch.number` -> `future`: i.e. all referenced L1 epochs must be there.
-    - `end_epoch_num == epoch.number`:
-      - If `batch.l1_origin_check != epoch.hash[:20]` -> `drop`: verify the batch is intended for this L1 chain.
-    - `end_epoch_num > epoch.number` -> `drop`: must have been duplicate batch,
-      we may be past this L1 block in the safe L2 chain.
+    - `start_epoch_num > epoch.number + 1` -> `future`:
```

Comment:
This condition is always false. 

Definitions:
`start_epoch_num = safe_l2_head.origin.block_number + (origin_changed_bit ? 1 : 0)`.
`epoch = safe_l2_head.l1_origin` (From [derivation specs](https://github.com/ethereum-optimism/optimism/blob/develop/specs/derivation.md?plain=1#L584))

Lets plug in each variable to the inequality.

`start_epoch_num > epoch.number + 1`
becomes
`safe_l2_head.origin.block_number + (origin_changed_bit ? 1 : 0) > safe_l2_head.l1_origin.number + 1`
which can never be satisfied.